shared/chat/audio/audio-recorder.native.tsx

```diff
@@ -314,7 +314,7 @@ const AudioButton = (props: ButtonProps) => {
           style={{
             position: 'absolute',
             right: 48,
-            top: 1,
+            top: -1,
```

Comment:
@mmaxim This recenters the icon inside the blue circle, but since I shrinked all the sizes of the message box icons, it's no longer aligned with the grey mic (we can see it on release). Do you know where to tweak the position for the whole mic + circles thing?

Reply:
All this stuff is absolute positioned, so if the sizes of this stuff changes you need to track down all the stuff that is off. I can do it tomorrow on this branch. Also I merged a change to the lock that conflicts with this (lock moves when you swipe up)